src/test/testing-library/AuthForm.test.tsx

```diff
@@ -127,6 +159,89 @@ describe('AuthForm', () => {
     fireEvent.click(privacyCheckbox);
     fireEvent.click(signUpButton);
 
-    await waitFor(() => expect(mockErrors[0].erorr).toBeDefined());
+    await waitFor(() => {
+      const serverErrorMessage: HTMLElement | null = getByRole(alertRole);
+
+      expect(serverErrorMessage).toBeInTheDocument();
+      expect(mockErrors[0].error).toBeDefined();
+    });
+  });
+
+  it('correct linkage between inputs and values', async () => {
+    const { getByPlaceholderText, getByRole } = render(
+      <MockedProvider addTypename={false}>
+        <AuthForm />
+      </MockedProvider>
+    );
+
+    const fullNameInput: HTMLInputElement = getByPlaceholderText(
```

Comment:
difficult to understand what is happening in this blocks of lines, need to simplify it